charts/gitops-server/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # FIXME check the app name
```

Comment:
is this a thing you wanted to fix now?

Reply:
negotiations are on going https://weaveworks.slack.com/archives/C03244W0C8H/p1647972249151359 :p 

Ideally yes but it's not a blocker on this and it may need a wider conversation